src/components/registration-view/registration-view.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Form, Button} from 'react-bootstrap';
+import './registration-view.scss';
+
+export function RegistrationView(props) {
+  const [ username, setUsername ] = useState('');
+  const [ email, setEmail ] = useState('');
+  const [ birthday, setBirthday ] = useState('');
+  const [ movies, setMovies ] = useState('');
+  const [ password, setPassword ] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(username, email, birthday, movies, password);
+    props.onLoggedIn(username);
+  };
+
+  return (
+    <Form className="registration-form">
+      <Form.Group controlId="formBasicUsername">
+        <Form.Label>Username</Form.Label>
+        <Form.Control 
+        type="text" placeholder="Enter Username" 
+        value={username} onChange={(e) => setUsername(e.target.value)}/>
+      </Form.Group>
+
+      <Form.Group controlId="formBasicEmail">
+      <Form.Label>Email</Form.Label>
+      <Form.Control 
+      type="text" placeholder="Enter Email" 
+      value={email} onChange={(e) => setEmail(e.target.value)}/>
+    </Form.Group>
+
+    <Form.Group controlId="formBasicBirthday">
+      <Form.Label>Birthday</Form.Label>
+      <Form.Control 
+      type="text" placeholder="Enter Birthday" 
+      value={birthday} onChange={(e) => setBirthday(e.target.value)}/>
+    </Form.Group>
+
+    <Form.Group controlId="formBasicMovies">
+      <Form.Label>Movies</Form.Label>
+      <Form.Control 
+      type="text" placeholder="Enter Movie Title" 
+      value={movies} onChange={(e) => setMovies(e.target.value)}/>
+    </Form.Group>
+
+    <Form.Group controlId="formBasicPassword">
+      <Form.Label>Password</Form.Label>
+      <Form.Control 
+      type="password" placeholder="Password" 
+      value={password} onChange={(e) => setPassword(e.target.value)}/>
+    </Form.Group>
+
+    <Button variant="primary" type="submit" onClick={handleSubmit}>
+      Register</Button>
+  </Form>
+
+ );
+}
+
+RegistrationView.PropTypes = {
+  endRegister: PropTypes.func.isRequired,
+}
```

Comment:
This little typo here, gives an error in the browser console. It should be `RegistrationView.propTypes = {...}`

Reply:
ok noted. I've changed it.